.github/workflows/build-test.yml

```diff
@@ -63,3 +67,15 @@ jobs:
 
       - name: Run audit
         run: yarn audit:ci
+
+  format:
+    name: "Format"
```

Comment:
I find this name deceptive because it only checks the format but doesn't format the files
can we change it to `Check Format`?

Reply:
The check is there to enforce that we are formatting the code locally, which we should already be doing and hopefully everyone is following [this](https://github.com/OffchainLabs/arb-token-bridge/blob/master/readme.md#development-tools). I think in the big majority of cases, there won't be anything to change.

If we were to format the code during CI, that means we'd have to somehow allow the CI to commit those changes back to the repository, which I don't think is safe.

Regarding the name, I'm gonna update that. 👍